src/Domain/CoordinateMaps/Frustum.cpp

```diff
@@ -39,7 +43,11 @@ Frustum::Frustum(const std::array<std::array<double, 2>, 4>& face_vertices,
                    lower_bound == -1.0 and upper_bound == 1.0 and
                    orientation_of_frustum_ == OrientationMap<3>{} and
                    not with_equiangular_map and projective_scale_factor == 1.0),
-      with_projective_map_(projective_scale_factor != 1.0) {
+      with_projective_map_(projective_scale_factor != 1.0),
+      sphericity_(sphericity) {
+  ASSERT(sphericity_ >= 0.0 and sphericity_ <= 1.0,
+         "The sphericity must be set between 0.0 (corresponding to a flat "
+         "surface) and 1.0 (corresponding to a spherical surface) inclusive.");
```

Comment:
Could you add in what the current value of the sphericity is in the message? I find it helpful when debugging to know what the wrong value actually is without having to go look it up.